### ISSUE TITLE
Include caching in action

### ISSUE THREAD

**Issue by koppor** [jbangdev/jbang-action#41]
It is now "modern" to include caching the action itself. See for example the [gradle setup action](https://github.com/gradle/actions/blob/main/docs/setup-gradle.md).

Thus, it would be convenient if the action also included automatic caching and not requiring the user to add a "cache" step for themselves (https://registry.hub.docker.com/r/jbangdev/jbang-action).

**Comment by maxandersen** [comment]
This would probably make more sense to do in https://github.com/jbangdev/setup-jbang - wdyt?

**Comment by koppor** [comment]
I agree - and I take this action then as "not recommended for use in general case", but users should use the other action.

Posted https://github.com/jbangdev/setup-jbang/issues/4
